tests/integration/internal/bootstrap.go

```diff
@@ -80,7 +80,7 @@ var _ = BeforeEach(func() {
 	l := logrus.New()
 	l.Out = GinkgoWriter
 	l.Level = logrus.DebugLevel
-	ctx = logging.ContextWithLogger(ctx, logging.NewLogrus(l))
+	ctx = logging.ContextWithLogger(ctx, logging.NewLogrusWithJsonFormatter(l))
```

Comment:
Not required here i think. That's integration testing, json is not easy to read ^^